sbi/inference/abc/smcabc.py

```diff
@@ -103,10 +106,15 @@ def __call__(
         num_simulations: int,
         epsilon_decay: float,
         distance_based_decay: bool = False,
-        ess_min: float = 0.5,
+        ess_min: float = None,
```

Comment:
Should be `Optional[float]` and would suggest to document that `None` turns it off